source/index.html.md

```diff
@@ -521,112 +562,191 @@ XML tag <OrgHeader></OrgHeader> wraps up all the organization related details su
-            <CompanyNameOverride></CompanyNameOverride>
-            <Address1>TEST ADDRESS1</Address1>
-            <Address2>TEST ADDRESS2</Address2>
-            <City>TESTCITY</City>
-            <State></State>
-            <PostCode></PostCode>
+            <PK>d8583f8c-1408-4c05-8498-efb171e3ce63</PK>
+            <CompanyNameOverride>EXAMPLE COMPANY ECNO</CompanyNameOverride>
+            <Address1>EXAMPLE ADDRESS A</Address1>
+            <Address2>EXAMPLE ADDRESS B</Address2>
+            <City>MIDDLESEX</City>
+            <State>HNS</State>
+            <PostCode>TW6 3JS</PostCode>
             <Phone></Phone>
-            <Fax></Fax>
-            <Mobile></Mobile>
+            <Fax />
+            <Mobile />
             <IsActive>true</IsActive>
-            <Email></Email>
-            <RelatedPortCode TableName="RefUNLOCO">
-              <Code>CNFZH</Code>
-              <PK>a47bc3d1-cf72-4ad9-a538-e3c4poklhnk</PK>
+            <Email>exampleemail@test.com</Email>
+            <RelatedPortCode TableName="ExampleUNLOCO">
+              <Code />
             </RelatedPortCode>
-            <CountryCode TableName="RefCountry">
-              <Code>CN</Code>
-              <PK>21eaa7e3-9009-4e17-9d96-f72d36iloko</PK>
+            <CountryCode TableName="ExampleCountry">
+              <Code>GB</Code>
             </CountryCode>
           </OrgAddress>
         </OrgAddressCollection>
         <OrgCompanyDataCollection>
           <OrgCompanyData Action="MERGE">
-            <PK>142dc66e-e946-4ffa-95a5-4c5ikikiplo</PK>
+            <APCategory />
+            <APPaymentTermDays>0</APPaymentTermDays>
+            <APPaymentTerms>COD</APPaymentTerms>
+            <ARConsolidatedAccountingCategory />
             <IsDebtor>false</IsDebtor>
-            <IsCreditor>false</IsCreditor>
+            <IsCreditor>true</IsCreditor>
+            <APCreditorGroup TableName="OrgCreditorGroup">
+              <Code>AIR</Code>
+            </APCreditorGroup>
+            <ControllingBranch TableName="GlbBranch">
+              <Code>LHR</Code>
+            </ControllingBranch>
+            <GlbCompany>
+              <Code>LHR</Code>
+            </GlbCompany>
+            <APDefltCurrency TableName="RefCurrency">
+              <Code>GBP</Code>
+            </APDefltCurrency>
           </OrgCompanyData>
         </OrgCompanyDataCollection>
-        <ShippingLine TableName="RefShippingLine" />
       </OrgHeader>
     </Organization>
   </Body>
 </Native>
-  </Body>
-</UniversalInterchange>
 ```
 
-#### Container reference data:
-This is a <UniversalInterchange> request.
-XML tag <UniversalShipment></UniversalShipment> wraps up all the container reference data,
-You can specify multiple containers by repeating the <Container></Container> XML tag.
-Similarly, multiple shipments can be specified by repeating <SubShipment></SubShipment> XML tag.
+#### **Container reference data**
+
+This is a **UniversalInterchange** request.
```

Comment:
Just a thought, but I think keeping the tags might be worth it contextually?
```suggestion
This is a `<UniversalInterchange>` request.
```
Backticks should highlight it.

Reply:
I thought this across all the other places to be honest, not entirely important anyway.

Reply:
Good point, Yes i wasn't sure i could do that.. will fix it.